### DISCUSSION TITLE
绘制工具、量算工具默认折叠使用哪个参数？

### DISCUSSION THREAD

**Discussion by kankantian** [zouyaoji/vue-cesium#464] [Q&A]
绘制工具、量算工具默认折叠使用哪个参数

**Comment by zouyaoji** [comment]
你好，这儿看：
https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/defaultOpts.ts

**Reply by kankantian** [reply]
你好，要是知道的话，尽情的告诉我吧！

**Reply by zouyaoji** [reply]
哦 不好意思 看错问题了，问的是默认折叠啊。

**Reply by zouyaoji** [reply]
麻烦参考这个：
https://github.com/zouyaoji/vue-cesium/discussions/175

**Reply by kankantian** [reply]
get it, thanks
